src/features/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { StyledNavBar, StyledNavText, StyledNavLink, StyledLogoText, StyledNavBarItems } from './Navigation.styles';
+import Logo from './logo.svg';
```

Comment:
to make the logo work I guess you need to  use it like this

<img src={Logo} />